application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -0,0 +1,213 @@
+                event.getGuild().getRolesByName("COMMUNITY AMBASSADOR", true);
+
+        for (Role role : userRoles) {
+            if (role.getPosition() >= communityAmbassador.get(0).getPosition()) {
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+
+    @Override
+    public void onMessageContext(MessageContextInteractionEvent event) {
+
+        if (!hasRequiredRoles(event)) {
+            event.reply("You're not authorised to use this command").setEphemeral(true).queue();
+        }
+
+        String originalMessage = event.getTarget().getContentRaw();
+
+
+        originalMessageId = event.getTarget().getId();
+        originalChannelId = event.getChannel().getId();
+
+        String author = event.getTarget().getAuthor().getId();
+
+
+        TextInput transferQuestionTitle =
+                TextInput.create(TRANSFER_QUESTION_TITLE_ID, "Title", TextInputStyle.SHORT)
+                    .setMaxLength(70)
+                    .setMinLength(8)
+                    .setValue(createTitle(originalMessage))
+                    .build();
+
+        TextInput transferQuestionInput = TextInput
+            .create(TRANSFER_QUESTION_INPUT_ID, "Transfer question menu", TextInputStyle.PARAGRAPH)
+            .setValue(originalMessage)
+            .setRequiredRange(3, 1000)
+            .build();
+
+        TextInput transferQuestionTag = TextInput
+            .create(TRANSFER_QUESTION_TAG, "Transfer question tags", TextInputStyle.SHORT)
+            .setValue("Java")
+            .build();
+
+
+
+        String transferQuestionModalComponentID = generateComponentId(author);
+        Modal reportModal = Modal.create(transferQuestionModalComponentID, "transfer question menu")
+            .addActionRow(transferQuestionTitle)
+            .addActionRow(transferQuestionInput)
+            .addActionRow(transferQuestionTag)
+            .build();
+
+        event.replyModal(reportModal).queue();
+    }
+
+
+    @Override
+    public void onModalSubmitted(ModalInteractionEvent event, List<String> args) {
+        User author = event.getJDA().getUserById(args.get(0));
+        createForumPost(event, author);
+
+    }
+
+    private static String createTitle(String message) {
+        if (message.length() >= TITLE_MAX_LENGTH) {
+            int lastWordEnd = message.lastIndexOf(' ', TITLE_MAX_LENGTH);
+
+            if (lastWordEnd == -1) {
+                lastWordEnd = TITLE_MAX_LENGTH;
+            }
+
+            message = message.substring(0, lastWordEnd);
+        }
+
+        return isTitleValid(message) ? message : "Untitled";
+    }
+
+    public static boolean isTitleValid(CharSequence title) {
+        String titleCompact = TITLE_COMPACT_REMOVAL_PATTERN.matcher(title).replaceAll("");
+
+        return titleCompact.length() >= TITLE_COMPACT_LENGTH_MIN
+                && titleCompact.length() <= TITLE_COMPACT_LENGTH_MAX
+                && !titleCompact.toLowerCase(Locale.US).contains("help");
```

Comment:
```suggestion
                && !titleCompact.toLowerCase(Locale.US);
```

Reply:
that suggestion doesnt make sense to me

Reply:
ohh sorry i meant to remove  the whole thing, coz we don't have the rule to discard  questions with "help" in their title  anymore right?

Reply:
yeah. shouldnt be blocked, i guess

Reply:
removed `!titleCompact.toLowerCase(Locale.US).contains("help")`